### ISSUE TITLE
NVDA doesn't read the subscribe button on youtube.com

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2364]
**Reported by ashleycox on 2012-05-20 10:17**
NVDA doesn't read the subscribe button on youtube.com, even though it is labeled. NVDA simply reads "button". The subscribe button is located underneath the users name, underneath the main heading of the page when you're viewing a video or channel. The label of the button changes - if you're not subscribed to a channel, it reads "subscribe", and if you are it reads "subscribed"... this functionality used to work, so I'm not sure what changed it.

**Comment by dkager** [comment]
Works for me in IE11 with current version of youtube.com.
